addons/event/i18n/event.pot

```diff
@@ -2326,12 +2333,6 @@ msgstr ""
 msgid "New"
 msgstr ""
 
-#. module: event
-#: model:ir.model.fields,field_description:event.field_event_event__activity_calendar_event_id
-#: model:ir.model.fields,field_description:event.field_event_registration__activity_calendar_event_id
-msgid "Next Activity Calendar Event"
```

Comment:
Same here, maybe it is outdated but we can keep it :)

Reply:
Thanks for explanation, I'll only add the new changes then :D